packages/core/src/generators/angular/index.ts

```diff
@@ -327,12 +319,18 @@ export const blockToAngular = (
   } else if (json.name === 'Show') {
     let condition = json.bindings.when?.code;
     if (condition?.includes('typeof')) {
-      let wordAfterTypeof = condition.split('typeof')[1].trim();
+      let wordAfterTypeof = condition.split('typeof')[1].trim().split(' ')[0];
       condition = condition.replace(`typeof ${wordAfterTypeof}`, `useTypeOf(${wordAfterTypeof})`);
     }
     str += `<ng-container *ngIf="${condition}">`;
     str += json.children.map((item) => blockToAngular(item, options, blockOptions)).join('\n');
     str += `</ng-container>`;
+    // else condition
+    if (isMitosisNode(json.meta?.else)) {
+      str += `<ng-container *ngIf="!(${condition})">`;
+      str += blockToAngular(json.meta.else, options, blockOptions);
+      str += `</ng-container>`;
+    }
```

Comment:
I don't mind this workaround. 

If we realize it has limitations down the line, we can improve it so it generates:

```suggestion
    const elseConditionName = /*some unique identifier*/;

    str += `<ng-container *ngIf="${condition}; else ${elseConditionName}">`;
    str += json.children.map((item) => blockToAngular(item, options, blockOptions)).join('\n');
    str += `</ng-container>`;
    // else condition
    if (isMitosisNode(json.meta?.else)) {
      str += `<ng-template #${elseConditionName}>`;
      str += blockToAngular(json.meta.else, options, blockOptions);
      str += `</ng-template>`;
    }
```

Reply:
yes @samijaber this should be the best way to go